src/modules/storage/LedgerStorage.ts

```diff
@@ -228,10 +228,10 @@ export class LedgerStorage extends Storages
 
         CREATE TABLE IF NOT EXISTS marketcap (
             last_updated_at     INTEGER NOT NULL,
-            price           BIGINT(20) UNSIGNED NOT NULL,
-            market_cap      BIGINT(20) UNSIGNED NOT NULL,
-            vol_24h         BIGINT(20) UNSIGNED NOT NULL,
-            change_24h      BIGINT(20),
+            price               DOUBLE  NOT NULL,
```

Comment:
https://github.com/bosagora/stoa/blob/v0.x.x/docs/Database_Schema.md
Please update the document.